pkg/controller/certificaterequests/sync.go

```diff
@@ -145,12 +147,17 @@ func (c *Controller) Sync(ctx context.Context, cr *v1alpha2.CertificateRequest)
 	crCopy.Status.CA = resp.CA
 
 	// invalid cert
-	_, err = pki.DecodeX509CertificateBytes(crCopy.Status.Certificate)
+	issuedCert, err := pki.DecodeX509CertificateBytes(crCopy.Status.Certificate)
 	if err != nil {
 		c.reporter.Failed(crCopy, err, "DecodeError", "Failed to decode returned certificate")
 		return nil
 	}
 
+	if issuedCert.NotAfter.Before(c.clock.Now()) {
+		c.reporter.Failed(crCopy, fmt.Errorf("NotAfter is %s while the time is %s", issuedCert.NotAfter.UTC().Format(expiryTimeFormat), c.clock.Now().UTC().Format(expiryTimeFormat)), "MissIssued", "Certificate is already expired")
```

Comment:
What's the reason for formatting the time here?

Also I think "MissIssued" is a misnomer. It may be exactly what we requested for.